Story/Episode 1/st_010180.csv

```diff
@@ -19,7 +19,7 @@ st_010180_0160#0,"""...Mi nombre es... Matoi."""
 st_010180_0170#0,"""No hay coincidencias en la base de datos. Al menos no en la base de datos ARKS..."""
 st_010180_0180#0,"""¿Podría ser de otro planeta? a primera vista, pensé que ella era un miembro de ARKS."""
 st_010180_0190#0,"""Matoi, ¿De donde eres? ¿En qué planeta vives?"""
-st_010180_0200#0,"""……う... ...<br>……<%me>"""
+st_010180_0200#0,"""......う... ...<br>……<%me>"""
```

Comment:
SIGH....

if you LEFT the string as unchanged, the patch file will take the English string `...Ah ...<%me>.`, because of our fallback
but now that you had changed, it will show up as `......う... ...<br>……<%me>`